features/contract-state/Totals.tsx

```diff
@@ -87,9 +113,12 @@ const Totals = () => {
     prettyTotalTokens: string = defaultMissingDataDisplay,
     prettyCollSymbol: string = "",
     prettyTokenSymbol: string = "",
-    getExchangeLinkCollateral: string = "https://app.uniswap.org/#/swap",
-    getExchangeLinkToken: string = "https://app.uniswap.org/#/swap",
-    exchangeName: string = "Uniswap"
+    collAddress: string = "",
+    tokenAddress: string = "",
+    getExchangeLinkCollateral: string = "",
+    getExchangeLinkToken: string = "",
+    exchangeName: string = "Uniswap",
+    addTokenToMetamask: any = null
```

Comment:
I wonder if instead of `any` you could just make it of type `() => void;`

Reply:
this threw an error when I tried. only config I could get ts to pass with is the code here.